src/iam.ts

```diff
@@ -22,7 +22,7 @@ import {promisifyAll} from '@google-cloud/promisify';
 import arrify = require('arrify');
 import {CallOptions} from 'google-gax';
 
-import {google} from '../proto/iam';
+import {google} from '../protos/protos';
 
```

Comment:
what does this `iam.ts`do? some exported type like `SetPolicyResponse` can be found in `protos.google.iam.v1.IAMPolicy`. Thanks!